mubench.reviewsite/tests/CSVHelperTest.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+use Monolog\Logger;
+use PHPUnit\Framework\TestCase;
+use MuBench\ReviewSite\Model\ExperimentResult;
+use MuBench\ReviewSite\Model\DetectorResult;
+use MuBench\ReviewSite\Model\Detector;
+use MuBench\ReviewSite\Model\Misuse;
+use MuBench\ReviewSite\Model\Review;
+
+use \MuBench\ReviewSite\CSVHelper;
+
+class CSVHelperTest extends TestCase
+{
+
+    private $csv_helper;
+    private $detector1;
+    private $detector2;
+    private $no_reviews_misuse;
+    private $positive_reviews_misuse;
+    private $resolved_review_misuse;
+
+
+    function setUp()
+    {
+        $this->csv_helper = new CSVHelper();
+        $this->detector1 = new Detector("-d1-", 1);
+        $this->detector2 = new Detector("-d2-", 2);
+        $positive_review = new Review([
+            'name' => '-reviewer1-',
+            'comment' => '-comment-',
+            'finding_reviews' => [
+                [
+                    'decision' => 'Yes',
+                ]
+            ]
+        ]);
+        $this->no_reviews_misuse = new Misuse(
+            ["misuse" => "0"],
+            [0 => []
+            ],
+            []
+        );
+        $this->positive_reviews_misuse = new Misuse(
+            ["misuse" => "0"],
+            [0 => []],
+            [$positive_review,
```

Comment:
Insert a newline before `$positive_review`, so that it is one the same level as the `new Review` below.